grid/social-media/followers/profile.supr

```diff
@@ -51,6 +51,32 @@ usecase GetFollowers safe {
       """
       imageUrl string
     } ]
+    """
+    Rate limit
+    Rate limit details.
+    """
+    rateLimit {
+      """
+      Total requests available
+      Total requests available in the time window.
+      """
+      totalRequestsAvailable number
+      """
+      Remaining requests available
+      Remaining requests available in the time window.
+      """
+      remainingRequestsAvailable number
+      """
+      Remaining requests available in percents
+      Remaining requests available in the time window in percents.
+      """
+      remainingRequestsAvailableInPercent number
+      """
+      Reset date time
+      Date and time when the rate limits will reset (in ISO 8601 format). Available only for providers without rolling rate limit window.
+      """
+      resetDateTime string
```

Comment:
You will receive this as Unix timestamp which, AFAIK, we can't easily format to ISO date in map (heads up @TheEdward162). I'd just pass the timestamp (converted from string to number):
```suggestion
      resetTimestamp number
```

Reply:
changed